File: 0242-valid-anagram/0242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        if(s.length()!=t.length()) return false;
        int[] res=new int[26];
        for(char ch:s.toCharArray()){
            res[ch-'a']++;
        }
        for(char ch:t.toCharArray()){
            res[ch-'a']--;
        }
        for(int i=0;i<26;i++){
            if(res[i]!=0){
                return false;
            }
        }
        return true;
    }
}

File: 0242-valid-anagram/0242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        if(s.size()!=t.size()) return false;
        int arr[26]={};
        for(int i=0;i<s.size();i++){
            arr[s[i]-'a']++;
            arr[t[i]-'a']--;
        }
        for(int x:arr){
            if(x!=0){
                return false;
            }
        }
        return true;
    }
};